src/scripts/fix_workspace_info.py

```diff
@@ -0,0 +1,85 @@
+"""
+Fixes workspace info to do the following.
+1. Make sure the "narrative" metadata field contains an int that points to the narrative.
+2. Make sure the "narrative_nice_name" metadata field is correct.
+3. Make sure the "is_temporary" metadata field exists and is correct.
+4. Adds a count of the number of narrative cells.
+5. Does nothing at all if there's > 1 narrative in the workspace.
+Note that while this fetches the Narrative object, it doesn't modify it in any way.
+"""
+from biokbase.workspace.client import Workspace
+
+ws_url = "https://ci.kbase.us/services/workspace"
+
+def fix_workspace_info(ws_id, token, verbose=False):
```

Comment:
Note there's no error handling for if the ws throws an error.

Reply:
That should work in to the calling function. If this is being called by some kind of shell script, then that should catch it when this crashes (and I'll change this to take in the url as a parameter or something). If we want this here to be an all-in-one script, then whatever manages the loop should deal with it.